rest-api/src/main/java/com/rednavis/webflux/demo/api/FileController.java

```diff
@@ -0,0 +1,40 @@
+package com.rednavis.webflux.demo.api;
+
+
+import org.springframework.http.MediaType;
+import org.springframework.http.codec.multipart.FilePart;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Schedulers;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+@RestController
+@RequestMapping("/upload")
+public class FileController {
+    @PostMapping(value = "/")
+    public Flux<String> upload(@RequestPart("file") Flux<FilePart> file) throws IOException {
```

Comment:
Please store temp file using suggestions from https://stackoverflow.com/questions/26860167/what-is-a-safe-way-to-create-a-temp-file-in-java

Reply:
Please add comments to that API endpoint: how it works, what's happen etc.